### DISCUSSION TITLE
Preset weight do not working

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1534] [Proposals]
### Goal

Não conseguimos incluir pedo medium em tema Vivo New

![image](https://github.com/Telefonica/mistica-design/assets/123083886/c163e035-330e-4b1c-8654-22761687f5ff)

https://tinyl.io/9dHR

### Context

Ao inserirmos no playroom as configurações de peso elas não funcionam no Vivo New.

### Impact

está diferente do conceito e do figma.

### Discussion Questions

-

**Comment by aweell** [comment]
Hola @BrunoBiagioniNeto,

La configuración de fuentes para storybook y playroom estaba asignando bold únicamente a los pesos 700 y no a los 500, el equivalente a medium para el sistema.

Should be solved with: https://github.com/Telefonica/mistica-web/pull/923
